windows/README.md

```diff
@@ -0,0 +1,77 @@
+# The Flutter Windows Desktop Embedder
+
+This framework provides the basic functionality for embedding a Flutter app on the Windows desktop.
+
+This README assumes you have already read [the top level README](../README.md), which contains important information and setup common to all platforms.
```

Comment:
Prefer wrapping lines to 80 characters when possible; that's the way the other READMEs are set up.